Libraries/Text/Text/RCTTextShadowView.m

```diff
@@ -218,7 +218,8 @@ - (NSTextStorage *)textStorageAndLayoutManagerThatFitsSize:(CGSize)size
     return cachedTextStorage;
   }
 
-  NSTextContainer *textContainer = [[NSTextContainer alloc] initWithSize:size];
+  // `[NSLayoutManager usedRectForTextContainer]` not calculated correctly in some cases(For example, https://github.com/facebook/react-native/issues/24970 ), so here we add one point to the height of text conatiner.
+  NSTextContainer *textContainer = [[NSTextContainer alloc] initWithSize:CGSizeMake(size.width, size.height + 1.0)];
```

Comment:
Wu, this method is being used by `measure` function as well, so it creates positive feedback loop between drawing and measuring. Event if it works for some cases, I don't think it's correct.

The idea we discussed was about providing additional pixel during drawing, not measuring (otherwise it will just increase the height of all blocks by one pixel).

Reply:
Ahha, I don't realize that it also be called by measure function, I'll adjust it. :)